Documentation/building/linux-instructions.md

```diff
@@ -43,6 +43,9 @@ Then install the packages you need:
 
 You now have all the required components.
 
+For Fedora 23 you will need to install the following:
+`$ sudo dnf cmake install clang lldb-devel libunwind-devel lttng-ust-devel libuuid-devel`
+
```

Comment:
Can you please rephrase the section content to layout the commands for various platforms (Linux, Fedora, etc) in a tabular manner?